.changeset/brown-ties-march.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/abi-typegen": minor
+"fuels": minor
+"@fuel-ts/versions": minor
```

Comment:
@petertonysmith94 Didn't notice this in time, but shouldn't this have been a patch? 🤔

Reply:
Not worth the trouble, I guess.

Let's try to keep an eye on future occurrences.